tests/framework/helpers/HtmlTest.php

```diff
@@ -1501,24 +1501,31 @@ public function testActiveCheckbox($value, array $options, $expectedHtml)
      */
     public function validAttributeNamesProvider()
     {
-        return [
+        $data = [
             ['asd]asdf.asdfa[asdfa', 'asdf.asdfa'],
             ['a', 'a'],
             ['[0]a', 'a'],
             ['a[0]', 'a'],
             ['[0]a[0]', 'a'],
             ['[0]a.[0]', 'a.'],
-
-            // Unicode checks.
-            ['ä', 'ä'],
-            ['ä', 'ä'],
-            ['asdf]öáöio..[asdfasdf', 'öáöio..'],
-            ['öáöio', 'öáöio'],
-            ['[0]test.ööößß.d', 'test.ööößß.d'],
-            ['ИІК', 'ИІК'],
-            [']ИІК[', 'ИІК'],
-            ['[0]ИІК[0]', 'ИІК'],
         ];
+
+        if (getenv('TRAVIS_PHP_VERSION') !== 'nightly') {
+            $data = array_merge($data, [
+                ['ä', 'ä'],
+                ['ä', 'ä'],
+                ['asdf]öáöio..[asdfasdf', 'öáöio..'],
+                ['öáöio', 'öáöio'],
+                ['[0]test.ööößß.d', 'test.ööößß.d'],
+                ['ИІК', 'ИІК'],
+                [']ИІК[', 'ИІК'],
+                ['[0]ИІК[0]', 'ИІК'],
+            ]);
+        } else {
+            $this->markTestIncomplete("Unicode characters check skipped for 'nightly' PHP version.");
```

Comment:
Why?

Reply:
Can't understand. Regex '\w' doesn't work with these unicode characters as expected in PHP dev versions. Other characters are OK.